web/src/nodes/helpers/node-ui.ts

```diff
@@ -35,21 +35,21 @@ const nodeColours = (name: string) => ({
 // prettier-ignore
 const nodeTypeUIMap: Partial<Record<NodeType, NodeTypeUI>> = {
   Admonition:       { ...stencilaIcon('admonition'),           ...nodeColours('violet')  },
-  Claim:            { ...shoelaceIcon('postage'),              ...nodeColours('fuchsia') },
-  CodeBlock:        { ...shoelaceIcon('braces'),               ...nodeColours('violet')  },
+  Claim:            { ...shoelaceIcon('postage'),              ...nodeColours('orange') },
+  CodeBlock:        { ...shoelaceIcon('braces'),               ...nodeColours('fuchsia')  },
   CodeChunk:        { ...shoelaceIcon('braces-asterisk'),      ...nodeColours('fuchsia') },
   DeleteBlock:      { ...shoelaceIcon('dash-circle'),          ...nodeColours('red')    },
-  Figure:           { ...shoelaceIcon('image'),                ...nodeColours('fuchsia') },
-  ForBlock:         { ...shoelaceIcon('repeat'),               ...nodeColours('pink')    },
-  Heading:          { ...shoelaceIcon('type-h1'),              ...nodeColours('purple')  },
+  Figure:           { ...shoelaceIcon('image'),                ...nodeColours('yellow') },
+  ForBlock:         { ...stencilaIcon('for-block'),            ...nodeColours('pink')    },
+  Heading:          { ...stencilaIcon('heading'),              ...nodeColours('purple')  },
   IfBlock:          { ...stencilaIcon('if-block'),             ...nodeColours('amber')   },
   InsertBlock:      { ...shoelaceIcon('plus-circle'),          ...nodeColours('lime')    },
-  InstructionBlock: { ...stencilaIcon('instruction-block'),    ...nodeColours('yellow')  },
-  List:             { ...stencilaIcon('list'),                 ...nodeColours('orange')  },
-  MathBlock:        { ...shoelaceIcon('infinity'),             ...nodeColours('fuchsia') },
+  InstructionBlock: { ...stencilaIcon('instruction-block'),    ...nodeColours('blue')  },
+  List:             { ...stencilaIcon('list'),                 ...nodeColours('yellow')  },
+  MathBlock:        { ...stencilaIcon('math-block'),           ...nodeColours('fuchsia') },
   Paragraph:        { ...stencilaIcon('paragraph'),            ...nodeColours('green')   },
```

Comment:
Somewhere else (in Discord maybe) I proposed using more neutral colors such as grey or a blue for paragraphs since there will be a lot of them and they are less "special" than things like instructions etc

Reply:
Yep just caught that comment. ALso for the toggle, that's where I thought the 
<img width="32" alt="image" src="https://github.com/stencila/stencila/assets/16539995/78805a33-80ea-4ecb-ba35-751bece5894b">
icon would function for. Since it is showing information about the document blocks. Potential for the future to also have a section dedicated to overall document info.

Reply:
Do we have an idea on what blocks specifically we could consider more "common" or "normal"? Just trying to figure out where to draw the line.

Reply:
> The initial thought was to give most of the basic blocks that feint green as a base a it's one of Stencilas main brand colours, so having it sprinkled more commonly throughout the app would give it a bit more of a connection to the brand. Making it the blue could also work, since we can associate green with things like blocks that are associated with addition.

I think that's good reasoning. I'd lean towards using the blue tones for the most basic blocks and the green for suggested additions.

> Do we have an idea on what blocks specifically we could consider more "common" or "normal"? Just trying to figure out where to draw the line.

I suspect a rough order or common-ess would be: paragraphs, headings, lists, tables, figures. All the rest feel more "special" to me so definitely the more vibrant colors there.